src/components/Messages/MessageText/customEmoji.js

```diff
@@ -0,0 +1,30 @@
+import { store } from '../../../store';
+import { Livechat } from '../../../api';
+import render from 'preact-render-to-string';
+import { emojify } from 'react-emojione';
+
+function search(nameKey, myArray) {
+	for (let i = 0; i < myArray.length; i++) {
+		if (myArray[i].name === nameKey) {
+			return myArray[i];
+		}
+	}
+}
+
+export const customEmoji = (text) => {
+
+	const { emojis } = store.state;
+	text = emojify(text, { output: 'unicode' });
+	const message = text.split(/:/g);
+	const content = [];
+	for (let i = 0; i < message.length; i++) {
```

Comment:
You can create an `arrow function` const and assign its return to `content` const.